selftests/functional/test_basic.py

```diff
@@ -476,6 +476,17 @@ def test_invalid_python(self):
         self.assertIn('1-%s:MyTest.test_my_name -> TestError' % test,
                       result.stdout)
 
+    @unittest.skipIf(process.system("which /bin/read", ignore_status=True),
+                     "/bin/read not available.")
+    def test_read(self):
+        os.chdir(basedir)
+        result = process.run("./scripts/avocado run /bin/read", timeout=10,
```

Comment:
Basically one of the kvm-unit-tests was hangling, waiting for input, with the #1461 it FAILS immediately.